test-runner/adapters/direct_python_sdk/direct_module_api.py

```diff
@@ -8,48 +8,49 @@
 import json
 from ..print_message import print_message
```

Comment:
>.. [](start = 5, length = 2)

I really, really, really dislike relative imports that move up a directory. They make me very uncomfortable, I recommend using the full path.